src/a-st-ext.adoc

```diff
@@ -221,14 +221,17 @@ same RISC-V hart. This can be appropriate when the LR/SC sequence is
 used to implement a parallel reduction operation.
 ====
 
+[NOTE]
+====
 Software should not set the _rl_ bit on an LR instruction unless the
 _aq_ bit is also set, nor should software set the _aq_ bit on an SC
 instruction unless the _rl_ bit is also set. LR._rl_ and SC._aq_
 instructions are not guaranteed to provide any stronger ordering than
 those with both bits clear, but may result in lower performance.
```

Comment:
The second sentence in this paragraph must be normative.  The first sentence is a logical consequence of the second, and so _could_ be non-normative, but I don't think it improves the readability of the spec to make this change.  I would undo this change.

Reply:
I believe you're right in the context of RVWMO, but this statement may have normative meaning in other contexts.  (Those contexts might not exist yet, of course...)